__tests__/app.test.js

```diff
@@ -40,6 +40,28 @@ describe("GET", ()=>{
                 })
         })
     })
+    describe("/api/reviews/:review_id", ()=>{
+        it("should return status 200, and a review object containing specific properties", ()=>{
+            return request(app)
+                .get('/api/reviews/1')
+                .expect(200)
+                .then(({body} )=>{
+                    const review = body.review[0]
+                    expect(review).toEqual(
+                        expect.objectContaining({
+                            title: expect.any(String),
+                            designer: expect.any(String),
+                            owner: expect.any(String),
```

Comment:
we aren't making sure in this test that we are also getting back the expected values for review 1 though - there's no guarantee that we are getting back the right data for the specified review, only that it is a review object (this would pass if we got back the data for review 6 and not review 1 for example). We know what we should be getting back for this specific review from looking at our test data, so I'd refactor to expect the exact values associated with the review